service/ui.go

```diff
@@ -279,8 +279,8 @@ func (uis *UIServer) GetServiceApp() *gimlet.APIApp {
 	}
 
 	// GraphQL
-	app.AddRoute("/graphql").Handler(playground.Handler("GraphQL playground", "/graphql/query")).Get().Wrap(needsLogin)
-	app.AddRoute("/graphql/query").Handler(handler.NewDefaultServer(graphql.NewExecutableSchema(graphql.New())).ServeHTTP).Options().Post().Get().Wrap(needsLogin)
+	app.AddRoute("/graphql").Wrap(needsLogin, allowsCORS).Handler(playground.Handler("GraphQL playground", "/graphql/query")).Get()
```

Comment:
The needsLogin will still cause redirect problems when working with this locally. How are you planning on getting around that?

Reply:
For now `needsLogin` will just need to be removed when working with gql server locally. It's not ideal but will work in the short term. There is a ticket for authentication with gql server and the details will be flushed out then.